debian/ooniprobe-cli.service

```diff
@@ -0,0 +1,53 @@
+# OnionBalance systemd target
```

Comment:
@FedericoCeratto I would recommend to set `OONI_HOME` to `/var/lib/ooniprobe`. This seems to be the correct way of forcing `ooniprobe` to use a specific directory as home directory. This approach is better than just letting `ooniprobe` choose based on the value of `$HOME` in that we don't create a `.ooniprobe` subfolder.

Reply:
Done